conda_smithy/linter/rattler_linter.py

```diff
@@ -20,3 +25,45 @@
     "about",
     "extra",
 ]
+TEST_KEYS = {"script", "python"}
+
+
+def lint_recipe_tests(
+    recipe_dir: Optional[str],
+    test_section: Dict[str, Any],
```

Comment:
For rattler-build, this should be a list of dictionaries I believe.

Reply:
that's right!